src/Bonsai.ML.HiddenMarkovModels/LoadHMMFromPkl.bonsai

```diff
@@ -0,0 +1,120 @@
94101277, -79.35193649913, -79.3538024195476, -79.33901250438, -79.3538024052913, -4.27770214689655, -79.3537999739468, -79.3538024152413, -79.3537476832867, -79.3538001630013, -0.0590336585215668, -79.3538024194494, -15.1403292390908, -79.3538024195476], [-80.8141761499455, -80.8299171351327, -80.8321243984454, -79.1140751828713, -80.7513948366163, -2.46917174255065, -59.1530651603166, -80.8321243984454, -80.8321243984454, -45.6935849036213, -2.7323847350908, -80.8321243984454, -80.8321243984454, -80.832124389641, -0.16311012032014, -7.15177346756846, -80.8321242340716], [-79.4246034910015, -3.48334852232484, -79.4246034910015, -5.7737892503062, -2.93766284446338, -79.4246034909991, -79.336913429818, -79.4164695681632, -4.34565213609343, -3.36034672586263, -79.4246014668271, -79.4244070303921, -79.4246022597813, -3.36103501832537, -79.4246034902494, -0.18535102870502, -79.4246034910015], [-3.13408601573862, -80.8656106188802, -4.3248839327091, -80.8654855265545, -80.8656106208069, -80.1649555362992, -80.8655973607584, -80.8656106208069, -80.8656106208069, -80.8382022891591, -80.8655171006304, -80.8656106208069, -80.8656106208069, -80.8656106208069, -80.8555535140235, -80.8656106208068, -0.0584500177614161]], observation_means=[[2.05933262307724, 10.8204332645415], [68.7980115395418, 149.313879689245], [0.0113367588749664, 0.0808845676209286], [36.8854430152106, 135.261998572096], [97.9835708320062, 475.786905396804], [4.85126962230773, 23.6361472322517], [44.6355192653933, 49.5566309333179], [266.329727183569, 710.965268736304], [173.521982388192, 173.718928476886], [17.616028067975, 196.233369659563], [17.0875828299435, 70.9425248823598], [204.737788814457, 189.798442944057], [260.143513242843, 262.732194667054], [122.001566709437, 101.435491122701], [9.67037560981843, 42.5625437060252], [96.2721341933364, 273.356625327438], [0.605044362615348, 3.14081365313523]], observation_covs=[[[0.905758344483852, 0], [1.2373681195321, 4.4831763035586]], [[14.3996184698293, 0], [10.9077146872554, 38.9063669683094]], [[0.015436036882364, 0], [0.109020732271086, 0.0444526519767825]], [[8.62408922415511, 0], [23.396230710446, 33.1375162351934]], [[65.0846425847929, 0], [-28.8024157037375, 67.0423181141901]], [[1.82643624659801, 0], [-1.17971110603607, 9.14393736242647]], [[10.9198108801625, 0], [3.33008442316356, 22.7612245320485]], [[51.3305896755247, 0], [13.743041597268, 189.84256808673]], [[12.5414378446675, 0], [-15.4565754923402, 78.6949525926226]], [[8.58696987374004, 0], [9.38430390783284, 66.0305935687787]], [[6.0341314486845, 0], [-9.64384045715633, 21.4295441508927]], [[9.1823933029414, 0], [13.3851117087156, 57.213049685513]], [[28.9288965124962, 0], [31.0513047915833, 60.3716689888219]], [[18.062239316743, 0], [11.8053467734538, 44.8280733649598]], [[3.79541470022462, 0], [-7.87657635998183, 13.537109427599]], [[28.3235630671307, 0], [-20.844035765898, 49.090267281817]], [[0.435879792544103, 0], [1.50536883080018, 1.53287076550159]]])</py:Script>
```

Comment:
Are these numbers specific fixed initialization parameters, or the result of previews runs which were saved here?